### ISSUE TITLE
If there is no trailing comma then the last style is not converted from jss-to-css

### ISSUE THREAD

**Issue by sambhav-gore** [in19farkt/css-to-jss#3]
First of all many thanks for creating this. I just now found this and I am sure this is going to be a big helper. I constantly find myself copying styles from web tools and manually converting to jss syntax.

Now I have a minor bug to report:

If I don't have a trailing comma in jss after the last style then the last style is not converted to css.
For example following is the jss:
```
wrapper: {
    display: "flex",
    paddingTop: 10,
    marginTop: 20
  }
```

And this results in following css after conversion:
```css
wrapper: {
    display: flex;
    padding-top: 10px;
    marginTop: 20
  }
```

See the last property is not change. Adding a trailing comma results in last property converting as well.

Similarly when going from CSS to JSS it always adds a trailing comma.

**Comment by in19farkt** [comment]
Thanks for this issue. This happens because the comma and semicolon defines the format of the string (CSS or JSS). I'll think about how to fix it.

**Comment by in19farkt** [comment]
Fixed in 1.0.6

**Comment by sambhav-gore** [comment]
Wow, Thanks for super quick fix and release of this.
